plugins/inputs/system/linux_sysctl_fs.go

```diff
@@ -80,9 +89,10 @@ func (sfs *SysctlFS) Gather(acc telegraf.Accumulator) error {
 }
 
 func init() {
+
 	inputs.Add("linux_sysctl_fs", func() telegraf.Input {
 		return &SysctlFS{
-			path: "/proc/sys/fs",
+			path: path.Join(GetHostSys(), "/fs"),
```

Comment:
Are you saying it should be more like path.Join(GetHostProc(), "/sys/fs")?

I am not sure would HOST_SYS be of any use then?